qa/workunits/rbd/rbd_mirror.sh

```diff
@@ -188,6 +188,14 @@ for i in ${image2} ${image4}; do
   compare_images ${POOL} ${i}
 done
 
+testlog "TEST: snapshot rename"
+snap_name='snap_rename'
+create_snapshot ${CLUSTER2} ${POOL} ${image2} "${snap_name}_0"
+for i in {1..20}; do
```

Comment:
@dillaman bashism is detected:

http://qa-proxy.ceph.com/teuthology/trociny-2016-07-13_23:55:05-rbd-wip-mgolub-testing---basic-vps/313371/teuthology.log
```
2016-07-14T02:18:44.681 INFO:tasks.workunit.cluster1.client.mirror.vpm137.stderr:+ rename_snapshot cluster2 mirror test2 snap_rename_ snap_rename_{1..20}
2016-07-14T02:18:44.682 INFO:tasks.workunit.cluster1.client.mirror.vpm137.stderr:+ local cluster=cluster2
2016-07-14T02:18:44.682 INFO:tasks.workunit.cluster1.client.mirror.vpm137.stderr:+ local pool=mirror
2016-07-14T02:18:44.691 INFO:tasks.workunit.cluster1.client.mirror.vpm137.stderr:+ local image=test2
2016-07-14T02:18:44.691 INFO:tasks.workunit.cluster1.client.mirror.vpm137.stderr:+ local snap=snap_rename_
2016-07-14T02:18:44.691 INFO:tasks.workunit.cluster1.client.mirror.vpm137.stderr:+ local new_snap=snap_rename_{1..20}
2016-07-14T02:18:44.692 INFO:tasks.workunit.cluster1.client.mirror.vpm137.stderr:+ rbd --cluster cluster2 -p mirror snap rename test2@snap_rename_ test2@snap_rename_{1..20}
2016-07-14T02:18:44.760 INFO:tasks.workunit.cluster1.client.mirror.vpm137.stderr:rbd: renaming snap failed: (2) No such file or directory
```